script/tool/lib/src/common/plugin_command.dart

```diff
@@ -100,6 +101,26 @@ abstract class PluginCommand extends Command<void> {
     return _shardCount!;
   }
 
+  /// Returns the [GitDir] containing [packagesDir].
+  Future<GitDir> getGitDir() async {
```

Comment:
nit: This should be a property.

Reply:
Changed. I wasn't aware that properties could be async.